packages/gazelle_cli/lib/commands/delete/delete.dart

```diff
@@ -0,0 +1,60 @@
+import 'dart:io';
+import 'package:args/command_runner.dart';
+import 'package:cli_spin/cli_spin.dart';
+import 'delete_project.dart';
+
+/// CLI command to delete a Gazelle project.
+class DeleteCommand extends Command {
+  @override
+  String get description => "Deleting a Gazelle project";
+
+  @override
+  String get name => "delete";
+
+  /// Creates a [DeleteCommand].
+  DeleteCommand() {
+    argParser.addOption(
+      "name",
+      abbr: "n",
+      help: "The name of the project you want to delete.",
+    );
```

Comment:
I would still remove this option.
The `path` option is enough to suit our needs.

If `path` is defined, you try to delete the project at given `path`, otherwise, you try to delete
the project in the current path.

Reply:
The name option is deleted now.